setup.py

```diff
@@ -20,7 +20,14 @@
 
 def _get_cpu_count():
     if platform.system() != "Windows":
-        return os.cpu_count()
+        cpu_count = os.cpu_count()
+        try:
+            user_max_cores = int(os.getenv('MAX_BUILD_CORES', cpu_count))
+        except ValueError as e:
+            print ("MAX_BUILD_CORES needs to be set to an integer.")
+            raise e
```

Comment:
unless you mean to explicitly use variable `e`, you can simply write
```suggestion
        except ValueError:
            print ("MAX_BUILD_CORES needs to be set to an integer.")
            raise
```

Reply:
Many of our exceptions use e, so I don't think it's an issue to use it here. Maybe it would be a more helpful error message to include whatever e is in the log message though.